packages/mdc-textfield/README.md

```diff
@@ -149,210 +135,100 @@ See [here](icon/) for more information on using icons.
-Alternatively, if you would like to change the border radius for every instance of MDC Text Field uniformly, regardless
-of the variant, you can override $mdc-text-field-border-radius in Sass.
-
-#### `mdc-text-field-box-corner-radius($radius)`
-
-This mixin customizes the border radius for a Text Field Box.
-
-#### `mdc-text-field-textarea-corner-radius($radius)`
-
-This mixin customizes the border radius for a Text Field `textarea`.
-
-### Using the JS component
-
-MDC Text Field ships with Component / Foundation classes which are used to provide a full-fidelity
-Material Design text field component.
-
-#### Including in code
-
-##### ES2015
-
-```javascript
-import {MDCTextField, MDCTextFieldFoundation} from '@material/textfield';
-```
-
-##### CommonJS
-
-```javascript
-const mdcTextField = require('mdc-text-field');
-const MDCTextField = mdcTextField.MDCTextField;
-const MDCTextFieldFoundation = mdcTextField.MDCTextFieldFoundation;
-```
-
-##### AMD
-
-```javascript
-require(['path/to/mdc-textfield'], mdcTextField => {
-  const MDCTextField = mdcTextField.MDCTextField;
-  const MDCTextFieldFoundation = mdcTextField.MDCTextFieldFoundation;
-});
-```
-
-##### Global
-
-```javascript
-const MDCTextField = mdc.textField.MDCTextField;
-const MDCTextFieldFoundation = mdc.textField.MDCTextFieldFoundation;
-```
-
-#### Automatic Instantiation
-
-```javascript
-mdc.textField.MDCTextField.attachTo(document.querySelector('.mdc-text-field'));
-```
-
-#### Manual Instantiation
-
-```javascript
-import {MDCTextField} from '@material/textfield';
+#### Helper Text
 
-const textField = new MDCTextField(document.querySelector('.mdc-text-field'));
-```
+The helper text provides supplemental information and/or validation messages to users. It appears on input field focus
+and disappears on input field blur by default, or it can be persistent. 
+See [here](helper-text/) for more information on using helper text.
 
-#### Controlling ripple instantiation
+#### Leading and Trailing Icons
 
-When `MDCTextField` is instantiated with a root element containing the `mdc-text-field--box` class,
-it instantiates an `MDCRipple` instance on the element in order to facilitate the correct
-interaction UX for text field boxes as outlined in the spec. The way this ripple is instantiated
-can be controlled by passing a ripple factory argument to the constructor.
+Leading and trailing icons can be added to MDC Text Fields as visual indicators as well as interaction targets.
+See [here](icon/) for more information on using icons.
 
-```js
-const textFieldBoxEl = document.querySelector('.mdc-text-field--box');
-const textField = new MDCTextField(textFieldBoxEl, /* MDCTextFieldFoundation */ undefined, (el, foundation) => {
-  // Optionally do something with the element or the Ripple foundation...
-  return new MDCRipple(el, foundation);
-});
-```
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-text-field` | Mandatory
+`mdc-text-field__input` | Mandatory
```

Comment:
This can be removed. Seems like this table should just be for root class modifiers.